### ISSUE TITLE
`Js::from()`/`Js::encode()` stubs missing `@psalm-taint-specialize` cause cross-callsite taint pollution

### ISSUE THREAD

**Issue by alies-dev** [psalm/laravel-psalm-plugin#1007]
## Summary

The stubs for `Illuminate\Support\Js::from()` and `Illuminate\Support\Js::encode()` declare `@psalm-flow ($data) -> return` but omit `@psalm-taint-specialize`. Without specialization, psalm pools every callsite into one taint node: tainted input at any single caller poisons the return value at every other caller, producing a cascade of false-positive `TaintedUserSecret` / `TaintedSystemSecret` reports across unrelated code.

## File

`stubs/common/Support/Js.phpstub:27` (and `:45` for `encode()`)

```php
/**
 * @psalm-taint-escape html
 * @psalm-taint-escape has_quotes
 * @psalm-flow ($data) -> return
 */
public static function from($data, $flags = 0, $depth = 512) {}
```

## Reproduction

Run `vendor/bin/psalm-laravel analyze` on `filamentphp/filament@4.x`. 68 taint errors fire. Investigation shows:

1. One genuine flow: `decrypt($action->getArguments()['encrypted'])['recoveryCodes']` -> `implode()` -> `Js::from()` at `packages/panels/src/Auth/MultiFactor/App/Actions/SetUpAppAuthenticationAction.php:137`.
2. Trace then jumps via `Js::from#1 - packages/infolists/src/Components/TextEntry.php:343:57`, where the actual argument is `$tooltip instanceof Htmlable` (a `bool` literal, unrelated to the source).
3. Re-emerges as the tainted source at every other `<?= Js::from(...) ?>` sink in the repo: `Notification.php:361`, `Component.php:191-192`, `CanGenerateBadgeHtml.php:127/129`, `CanGenerateButtonHtml.php:191/193`, `CanGenerateDropdownItemHtml.php:110/112/134/136`, `CanGenerateIconButtonHtml.php:130/132`, `CanGenerateLinkHtml.php:149/151`, `SelectColumn.php:1024`, `TextInputColumn.php:279`. None of these sinks receive the decrypted value.

Per psalm docs (https://psalm.dev/docs/security_analysis/annotations/#psalm-taint-specialize), specialization is required for functions whose taint behavior should be tracked per-call-site rather than globally.

## Proposed fix

```diff
 /**
  * @psalm-taint-escape html
  * @psalm-taint-escape has_quotes
  * @psalm-flow ($data) -> return
+ * @psalm-taint-specialize
  */
 public static function from($data, $flags = 0, $depth = 512) {}
```

Same for `Js::encode()`.

## Impact

Eliminates ~60 of the 68 errors in the filament repro and unblocks per-callsite analysis for downstream users.

## Follow-up suggestion (separate issue if preferred)

Audit other stubs with `@psalm-flow` and no `@psalm-taint-specialize`. Likely candidates: `e()`, `Str::*`, `Arr::*` helpers, collection passthroughs. Same false-positive class.
